charts/fission-all/templates/_helpers.tpl

```diff
@@ -31,3 +31,13 @@ canary:
   {{- printf "\n" -}}
 {{- end -}}
 
+{{- define "fission-all-bundleImage" -}}
```

Comment:
Have you considered something simpler like this one?
```yaml
{{- define "fission-all-bundleImage" -}}
{{- if .Values.repository -}}
    {{ .Values.repository }}/{{ .Values.image }}:{{ .Values.imageTag }}
{{- else -}}
    {{ .Values.image }}:{{ .Values.imageTag }}    
{{- end }}
{{- end -}}
```

Reply:
Yes, I tried this initially but this wasn't working but I have just tried it and it is working. May be I was doing something wrong or missing something. Thanks for the suggestion. I'll change it.